client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -600,6 +600,14 @@ return 'Unknown';
             window['turf'].point([long, lat]),
             this.defaultZoomLevel,
           );
+        } else if (layerId.includes('protected-lands-access-restrictions')) {
+          this.identifyItem;
```

Comment:
How come this gets declared in the function? Won't it already be declared at the class level?

Reply:
sorry, typo. was using as debug purpose